### DISCUSSION TITLE
Collection NFT marked as scam

### DISCUSSION THREAD

**Discussion by andt99** [phantom/docs#272] [Help & Support] [answered]
### Summary

I have a collection on solana. How to make it not marked as scam and can be shown on extension / mobile app.

### Steps to Reproduce

I mint a NFT following the token metadata standard (my NFT example: https://explorer.solana.com/address/8phFXS8zWQ8zjwz24SqmZMPunsD49L3JfiB951rR1iv3). How to make it display on phantom wallet.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @andt99,

Are recipients opting in to receive this nft?

**Reply by andt99** [reply]
Yes, they are. They will claim themselves the NFT voucher through our program.

**Reply by adamdelphantom** [reply]
@andt99 Where is the site and social media accounts for this program?

**Reply by andt99** [reply]
Hi @adamdelphantom, below are the social media and the site hosting the program:
- https://x.com/RelendOfficial
- https://x.com/RenecBlockchain
- Site: https://lend.renec.org/dashboard

**Reply by adamdelphantom** [reply]
@andt99 Thank you for providing this! I will follow up shortly

**Reply by adamdelphantom** [reply, marked as answer]
should be good to go, now
